### PR TITLE
chore(datepicker): Add in vendor/assets

### PR DESCRIPTION
**Pull request by semia**
- Update datepicker file to use coffeescript
- Use 'ready turbolinks:load' instead of method: :get

Related to issue #125

### CODE REVIEW

File: vendor/assets/javascripts/components/datepicker.js.coffee [hunk 1]
```diff
@@ -1,8 +1,8 @@
-jQuery ->
-  $('#datepicker').datepicker({
+$(document).on 'ready turbolinks:load', ->
```
Comment (arnlen) [inline vendor/assets/javascripts/components/datepicker.js.coffee]: Great! 🎉

**Comment by arnlen** [comment]
The `vendor/assets` folder is dedicated to external libraries.
> ie: if we're using a Geoloc lib

In the case of the Datapicker, it's still part of our application main codebase.
Moreover, it's hightly coupled to our code, because of the `id` en `turbolink` reference:

```
$(document).on 'ready turbolinks:load', ->
  $('#datepicker').datepicker( [...]
```

A question we could ask to determine if this is inside or outside the scope of our application: if we remove this file by another one, would I need to update
- the new file?
- or my codebase?

If I need to update my codebase, it's probably an external lib.
If I need to update the file, then it's an internal lib.

**Comment by semia** [comment]
Oki thx so I move it to the original folder :  app/assets/javascript :-)

**Comment by arnlen** [comment]
Good one, thx! 👍

**Pull request merged**